roles/openstack-stack/templates/heat_stack.yaml.j2

```diff
@@ -142,18 +142,17 @@ resources:
 #            cluster_id: {{ stack_name }}
 #      public_key: {{ ssh_public_key }}
 
-{% if openstack_flat_secgrp|bool %}
-  flat-secgrp:
+  basic-secgrp:
```

Comment:
not really, all but dns. I tried to explain that both in commit msg and the description below